### PR TITLE
POL-921 Add Region Filtering to Azure Rightsizing Compute

### PR DESCRIPTION
**Pull request by XOmniverse**
### Description

This non-breaking update adds the ability to filter by region, similar to other Azure usage reduction policies. 

### Link to Example Applied Policy

https://app.flexera.com/orgs/6/automation/applied-policies/projects/7954?policyId=6509ac05656c530001e1762e
(Parameter is set to only allow instances in the eastus region, and this is reflected in the incident)

### Contribution Check List

- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable
- [X] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/rightsize_compute_instances/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.3
+
+- Added ability to filter resources by region
```
Comment (nia-vf1) [inline cost/azure/rightsize_compute_instances/CHANGELOG.md:5]: Is adding new parameters a breaking change and therefore this should be version 4.0?

cc - @bryankaraffa

Comment (XOmniverse) [reply cost/azure/rightsize_compute_instances/CHANGELOG.md:5]: @nia-vf1 Only if they don't have default values, and the default values don't enable it to function as it did before the parameter existed.

In this case, simply not specifying a value for the new parameters will result in the policy functioning identically to the previous version, so the change is not breaking.

Comment (nia-vf1) [reply cost/azure/rightsize_compute_instances/CHANGELOG.md:5]: @XOmniverse - You mean only if they do have default values?

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Pull request merged**